libbeat/docs/upgrading.asciidoc

```diff
@@ -1,160 +1,324 @@
+// detail, similar to what Kibana does: https://www.elastic.co/guide/en/kibana/master/upgrade-standard.html
+// WDYT?
+//
+// Besides copying over the config file (for all beats), and pointing to the 
+// registry file for Filebeat, what else needs to be preserved.
+
+[[upgrading-to-6.7]]
+==== Upgrade to {beats} 6.7 before upgrading to 7.0
 
-The upgrade procedure assumes that you have Beats version 5.6 installed. If you
-are on a previous 5.x version of Beats, please upgrade to version 5.6 first.
-The reason is that the Elasticsearch mapping template in 5.6 was modified to be
-compatible with Elasticsearch 6.0 (by removing the `_all` settings).
+The upgrade procedure assumes that you have {beats} 6.7 installed. If you're on
+a previous 6.x version of {beats}, upgrade to version 6.7 first. This is
+required because the {es} mapping template in 6.7 was modified to be compatible
+with {es} 7.0 (the `_type` setting changed from `doc` to `_doc`).
 
-For this reason, after upgrading to 5.6, you need to make sure that the 5.6
-template is loaded. You can do this by temporarily enabling the
-`output.elasticsearch.template.overwrite` setting, for example with Metricbeat:
+After upgrading to 6.7, make sure the 6.7 template gets loaded. You can do this
+by temporarily enabling the `output.elasticsearch.template.overwrite` setting,
+for example with Metricbeat:
 
 [source,shell]
-------------------------------------------------------------------------------
+----
 metricbeat -e -E output.elasticsearch.template.overwrite=true
-------------------------------------------------------------------------------
+----
 
-Alternatively, you can manually force loading the template:
+Or you can manually force loading the template:
 
 [source,shell]
-------------------------------------------------------------------------------
+----
 curl -XPUT -H'Content-Type: application/json'  http://localhost:9200/_template/metricbeat -d @metricbeat.template.json
-------------------------------------------------------------------------------
+----
+
+To check the template version, open the {kib} Console and run:
+
+[source,js]
+----
+GET /_template/metricbeat
+----
+// CONSOLE
 
-To check which version of the template is loaded, open Kibana Console, call `GET
-/_template/metricbeat`, and look for the version string. Note that you need to
-do this for each Beat type that you are running (e.g. Filebeat, Metricbeat,
-Packetbeat).
+Look for the version string in the response. Repeat this step for every Beat
+running in your environment.
 
+*Metricbeat and Filebeat users:* If you use {beats} central management,
+make sure you migrate the {beats} central management index to 6.7 before you
+upgrade to 7.0. 
+
+// REVIEWERS: Not sure I should mention this here ^^. I didn't link directly
+// to the 6.7 breaking changes docs here because I'm concerned about users
+// getting into the wrong upgrade docs and being confused.
```

Comment:
We can also point to the Beats blog post: https://www.elastic.co/blog/beats-6-7-0-released

The migration of CM to 6.7 is not really user friendly, but CM is not GA. The script is some means to help early testers/adopters with their setup without having to start over.

How about adding a disclaimer and/or contents from actual blog post here?